examples/maps/maps.bal

```diff
@@ -1,5 +1,10 @@
 import ballerina/io;
 
+type ClassMarks record {|
+    int sam;
+    int jon;
+|};
```

Comment:
How about changing this to

```suggestion
// A record representing marks received for humanities subjects.
type Humanities record {|
    int history;
    int geography;
|};
```

and for L105 doing

```ballerina
Humanities humanitiesMarks = {history: 80, geography: 75};
map<int> allMarks = {physics: 100, ...humanitiesMarks, chemistry: 75};
```

Reply:
Okay. Thanks :-)